src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -175,22 +326,26 @@ func startCluster(
 		return err
 	}
 
-	reqPorts := n * 5 // we need 5 ports per node
-	ports, err := vnet.GetFreePortPool(int(reqPorts))
-	if err != nil {
-		return err
-	}
-
 	// Start a seed node.
 	var (
-		seedID            uint
-		seedKafkaPort     = ports[0]
-		seedProxyPort     = ports[1]
-		seedSchemaRegPort = ports[2]
-		seedRPCPort       = ports[3]
-		seedMetricsPort   = ports[4]
+		seedID                                                         uint
+		seedKafkaPort                                                  = cPorts.KafkaPorts[0]
+		seedProxyPort, seedSchemaRegPort, seedRPCPort, seedMetricsPort uint
```

Comment:
Mine putting each of these on their own line to not have weird tabbing on the lines above?